PIC_undo_batch.py

```diff
@@ -157,9 +166,11 @@ def picturae_csv_undo(self, table: str, MD5: str):
         self.batch_tree_pruner(table=table,
                                timestamp1=start_time,
                                timestamp2=end_time)
+
         # clearing picbatch records
```

Comment:
picbatch records are only deleted if any actual table records are deleted, if the purger is run with no change, on an md5, there is probably an error in the config or md5.